x-pack/platform/plugins/shared/cases/server/client/cases/get.ts

```diff
@@ -172,7 +172,7 @@ export const get = async (
   clientArgs: CasesClientArgs
```

Comment:
Why not remove `includeComments` also from the cases client `GetParams`? Is it being used by other teams?

Reply:
Because we have no alternative approach yet https://github.com/elastic/kibana/issues/207797